CHANGELOG.md

```diff
@@ -3,7 +3,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
-### 🔄 Changed
+## StreamChat
+### ✅ Added
+- Pinning channels [#3518](https://github.com/GetStream/stream-chat-swift/pull/3518)
+  - Add `Chat.pin()` and `Chat.unpin()`
+  - Add `ChatChannelController.pin(completion:)` and `ChatChannelController.unpin(completion:)`
+  - Add `FilterKey.pinned` for filtering channel lists
+  - Add `ChannelListSortingKey.pinnedAt`
+  - Add `ChatChannel.membership.pinnedAt`
+  - Add `ChatChannel.isPinned`
```

Comment:
I think we can improve the API, since this action only makes sense for the current user:

```suggestion
- Pinning channels [#3518](https://github.com/GetStream/stream-chat-swift/pull/3518)
  - Add `ConnectedUser.pin(channelId:)` and `ConnectedUser.unpin(channelId:)`
  - Add `CurrentChatUserController.pin(channelId:)` and `CurrentChatUserController.unpin(channelId:)`
  - Add `FilterKey.pinned` for filtering channel lists
  - Add `ChannelListSortingKey.pinnedAt`
  - Add `ChatChannel.membership.pinnedAt`
  - Add `ChatChannel.isPinned`
```